src/__tests__/Search/__snapshots__/SearchResultsList.test.tsx.snap

```diff
@@ -434,9 +421,6 @@ exports[`SearchResultsList should match snapshot 1`] = `
                     </p>
                   </div>
                 </li>
-                <span
-                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
-                />
```

Comment:
I'm not 100% sure what caused these changes, but I believe it's due to an update of material ui